datanode/entities/aggregated_ledgerentries.go

```diff
@@ -18,20 +18,24 @@ type AggregatedLedgerEntries struct {
 	VegaTime     time.Time
 	Quantity     decimal.Decimal
 	TransferType *LedgerMovementType
-	PartyID      *PartyID
 	AssetID      *AssetID
-	MarketID     *MarketID
-	AccountType  *types.AccountType
+
+	AccountFromPartyID     *PartyID
+	AccountFromMarketID    *MarketID
+	AccountFromAccountType *types.AccountType
+	AccountToPartyID       *PartyID
+	AccountToMarketID      *MarketID
+	AccountToAccountType   *types.AccountType
```

Comment:
Super minor and a matter of opinion (e.g. non merge blocking) but i'd probably have just called these e.g. `FromPartyID`

Reply:
Do you mean `FromPartyId`, `ToPartyId`, `FromMarketId`, `ToMarketId`, `FromAccountType`, `ToAccountType`? 
If so, what about 
`SenderPartyId`, `ReceiverPartyId`, `SenderMarketId`, `ReceiverMarketId`?

Reply:
I like that

Reply:
All right, should i include that change now?

Reply:
it makes sense to do so

Reply:
All right the change is applied